### ISSUE TITLE
This warning is caused by a @providesModule declaration...

### ISSUE THREAD

**Issue by andrewimm** [facebook/react-vr#76]
On the first initialization of the server, you may see a stream of warning messages like this one:

```
This warning is caused by a @providesModule declaration with the same name across two different files.
jest-haste-map: @providesModule naming collision:
```

As the message notes, this is only a warning, and does prevent the server from running. The underlying issue was fixed in one of our dependencies, and should cease to be a problem once that package deploys its latest version.

**Comment by spikebrehm** [comment]
FWIW I started getting these warnings after upgrading from 0.1.6 to 0.2.0.

**Comment by andrewimm** [comment]
Update: underlying packager dependency has been updated and published. This should get resolved when we update to React Native 0.44
